### PR TITLE
adds logout button and functionality

### PR DESCRIPTION
**Pull request by dettma1**

### CODE REVIEW

File: src/main/java/codeu/controller/LogoutServlet.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  @Override
+  public void init() throws ServletException {
+    
+  }
+
+
+
+  /**
+   * This function fires when a user requests the /logout URL. It simply forwards the request to
+   * login.jsp.
```
Comment (AzeeSoft) [inline src/main/java/codeu/controller/LogoutServlet.java]: The method seems to be forwarding the request to index.jsp, and not login.jsp.

File: src/main/java/codeu/controller/LogoutServlet.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+    HttpSession session = request.getSession(false);
+    if(session != null)
+      session.invalidate();
+    request.getRequestDispatcher("/index.jsp").forward(request,response);
+
+  }
+
+
+  /*@Override
```
Comment (AzeeSoft) [inline src/main/java/codeu/controller/LogoutServlet.java]: If we are not expecting a POST request here, we could probably just remove this entire block comment.

File: src/main/webapp/WEB-INF/view/logout.jsp [hunk 3]
```diff
@@ -0,0 +1,31 @@
+<%--
```
Comment (AzeeSoft) [inline src/main/webapp/WEB-INF/view/logout.jsp]: I don't think this file is being used anywhere. This file could probably be removed.

Comment (dettma1) [reply src/main/webapp/WEB-INF/view/logout.jsp]: Yep, I was just using for it testing purposes!

**Pull request merged**